### PR TITLE
Support RHEL based OS using avalanchego installer

### PR DESCRIPTION
**Pull request by DegenComedian**
- tested on Rocky Linux 8.6 (binary compatible with RHEL 8.6)
- tested on Ubuntu 22 Server
- RHEL 8 comes with a SELinux setting, which does not allow running systemd processes from home dir, therefore a proper context has been set on the avalanchego executable

### CODE REVIEW

File: scripts/avalanchego-installer.sh [hunk 1]
```diff
@@ -363,6 +413,25 @@ create_config_file
 create_service_file
 chmod 644 avalanchego.service
 sudo cp -f avalanchego.service /etc/systemd/system/avalanchego.service
+
```
Comment (jpop32) [inline scripts/avalanchego-installer.sh:416]: Why this duplicate block? Forgotten scratch space?

File: scripts/avalanchego-installer.sh [hunk 2]
```diff
@@ -363,6 +413,25 @@ create_config_file
...
+foundIP="$(dig +short myip.opendns.com @resolver1.opendns.com)"
+foundArch="$(uname -m)"                         #get system architecture
+foundOS="$(uname)"                              #get OS
+if [ "$foundOS" != "Linux" ]; then
+  #sorry, don't know you.
+  echo "Unsupported operating system: $foundOS!"
+  echo "Exiting."
+  exit
+fi
+foundIP="$(dig +short myip.opendns.com @resolver1.opendns.com)"
```
Comment (jpop32) [inline scripts/avalanchego-installer.sh:426]: And once more?

**Review by jpop32 — CHANGES_REQUESTED** [review]
Please see inline comments.

**Comment by cgcardona** [comment]
@DegenComedian can you please address @jpop32's review comments? Thx

File: scripts/avalanchego-installer.sh (hunk 1 above)
Comment (DegenComedian) [reply scripts/avalanchego-installer.sh:416]: this was a copy and paste error. Duplicate block can be deleted.

File: scripts/avalanchego-installer.sh (hunk 2 above)
Comment (DegenComedian) [reply scripts/avalanchego-installer.sh:426]: this was a copy and paste error. Duplicate block can be deleted.

**Comment by cgcardona** [comment]
Closing in favor of https://github.com/ava-labs/avalanche-docs/pull/1102. TY!

**Pull request closed without merging**